### DISCUSSION TITLE
[Feature] The iOS App DOESN'T ASK for REMOVE PHOTOS after Uploaded it into Immich Cloud (Essential Feature Request)

### DISCUSSION THREAD

**Discussion by wackyesolution** [immich-app/immich#2635] [Feature Request]
### The feature

Google photos , in the settings give you the chance to clean all iOS local media libnrary from the media that is already uploaded on the cloud.
how I can do the same thing using Immich?

**I think it's an essential function because it allows you to clean the gallery with the automatic certainty that the media has been successfully uploaded.💚**

Someone can help me?

### Platform

- [X] Server
- [ ] Web
- [X] Mobile

**Comment by alextran1502** [comment]
Any mass removing/deleting feature will be added as the last thing before the stable release is reached

**Comment by bo0tzz** [comment]
Closing as duplicate of #165
